js/modules/Forms/RendererController.js

```diff
@@ -122,6 +122,9 @@ export class GlpiFormRendererController
 
             debouncedComputeItemsVisibilities();
         });
+
+        // Enable actions
+        $(this.#target).removeClass('pointer-events-none');
```

Comment:
This make sure users can't interact with the form until the JS is loaded (mostly helpful for automated testing like e2e where the test will click too fast on the page - unlikely to happens to a real user).